src/SDKs/Graph.RBAC/Graph.RBAC/Generated/GroupsOperationsExtensions.cs

```diff
@@ -132,37 +136,6 @@ public static async Task AddMemberAsync(this IGroupsOperations operations, strin
                 (await operations.AddMemberWithHttpMessagesAsync(groupObjectId, parameters, null, cancellationToken).ConfigureAwait(false)).Dispose();
             }
 
-            /// <summary>
```

Comment:
These are breaking changes. Please bump the minor version [here](https://github.com/Azure/azure-sdk-for-net/blob/b4aee1f3b3058845d0a2cec84628956ef8a4007a/src/SDKs/Graph.RBAC/Graph.RBAC/Microsoft.Azure.Graph.RBAC.csproj#L4)
Also update `PackageReleaseNotes` and `AssemblyInfo.cs`
If the REST API version is stable the nuget package version generated MUST also be stable
Please link the REST spec PR